spec/puppetserver/ca/action/generate_spec.rb

```diff
@@ -21,7 +19,7 @@
   subject { Puppetserver::Ca::Action::Generate.new(logger) }
 
   it 'prints the help output & returns 1 if invalid flags are given' do
-    exit_code = Puppetserver::Ca::Cli.run(['generate', '--hello'], stdout, stderr)
+    _, exit_code = subject.parse({ '--hello' => ''})
```

Comment:
I notice elsewhere we use parse with an array like: ['--hello', '']

Reply:
Oh right, I should change this. I remember there being a comment before about being surprised this works :P